plasma_framework/contracts/src/transactions/PaymentTransactionModel.sol

```diff
@@ -41,36 +39,47 @@ library PaymentTransactionModel {
      * @return A decoded PaymentTransaction struct
      */
     function decode(bytes memory _tx) internal pure returns (PaymentTransactionModel.Transaction memory) {
```

Comment:
Some really minor comments on the document. 
1. Can we change the beginning from `/*` to `/**`? This makes the IDE happy to change color :)
1. `tx` -> `_tx`